testutil/fuzzing/fuzz.go

```diff
@@ -0,0 +1,73 @@
+package fuzz
+
+// This file implements fuzzers using go-fuzz.
+// More information about go-fuzz can be found here:
+// https://github.com/dvyukov/go-fuzz
+
+// To run the fuzzer locally, follow these steps:
+// 1) go get github.com/influxdata/telegraf
+// 2) go get -u github.com/dvyukov/go-fuzz/go-fuzz
+// 3) go get -u github.com/dvyukov/go-fuzz/go-fuzz-build
+// 4) cd into dir of fuzz.go
+// 5) $GOPATH/bin/go-fuzz-build
+// 6) $GOPATH/bin/go-fuzz
+
+import (
+	"github.com/influxdata/telegraf/plugins/parsers/csv"
+	"github.com/influxdata/telegraf/plugins/parsers/json"
+	"time"
+)
+
+var DefaultTime = func() time.Time {
+	return time.Unix(3600, 0)
+}
+
+func FuzzCsvParse(data []byte) int {
```

Comment:
I am not a Telegraf contributor, but I see a lot of parsers in Telegraf (https://github.com/influxdata/telegraf/tree/master/plugins/parsers). I am afraid that the proposed organization of the fuzzers will make this file grow to a large size. Can we reorganize so that e.g. the CSV parser fuzzer code is in the correct directory for the CSV parser?